### PR TITLE
a few new child notes and video snippets.

### PR DESCRIPTION
**Pull request by DoctorBoomstick**
#### Summary
Content "Adds a collection of new snippets, both child notes and E-logs."
#### Purpose of change
More variety in present snippets.
#### Describe the solution
Adds 10 new snippets: 2 child notes and 8 e-logs, with the latter being quite sizable.
#### Describe alternatives you've considered
Not adding anything, or trying to keep the snippets shorter, the latter of which I have already tried to do to the best of my ability without jeopardizing the meaning behind the snipits.
#### Testing
They’re simple text additions.

### CODE REVIEW

**Comment by sparr** [comment]
test failure is related to json not changed by this PR

File: data/json/snippets/e_logs.json [hunk 1]
```diff
@@ -74,24 +74,56 @@
...
...n the hospital after the scuffle in <city> last week, and we wanted to say that we're actually really missing your broken ass out here in bum fuck nowhere. We're recording this video cause we wanted to tell you...\"\n\n\"GET WELL SOON!\" As the raucous chorus of disjointed good wishes dies down, the friends' grins rapidly melt away and are replaced by a look of unified confusion as the call... repeats.  An uneasy silence falls across the group as, from an unseen point in the darkness, the chant resounds, growing more distorted and twisted with each inhuman repetition.  A muttered \"What the fuck?\" can be heard as the call continues, seeming to grow louder or closer, now accompanied with the audible crunch of trampled vegetation.  Nervously, a young man draws a 10mm pistol from beneath his jacket and opens his mouth, before a shapeless, pink mass gallops from the gloom behind him on many nimble legs, rapping mauve tendrils about his head as it grips his shoulders with twitching claws, before wrenching its horrific tendrils and ripping the man’s head clean off.\n\n\"GET WELL SOON!\" screams the Mi-go, blood showering from its fleshy, flower-like head before it lunges at the camera."
```
Comment (Daved27hundred) [inline data/json/snippets/e_logs.json:107]: add "GET WELL SOON!" to the string of lines migo can say ingame, as a reference to this snippet.

**Comment by Inglonias** [comment]
This is some well written stuff. Good job.

File: data/json/snippets/children_notes.json [hunk 2]
```diff
@@ -137,5 +137,13 @@
         "text": "\"In case of emergency fill out this form so that you may present it to emergency personnel for identification purposes.  Keep this form with your emergency supplies.  If You need to request new copies of this form, speak with the school safety office.  Remember to keep calm, and stay in one location if possible, this will help emergency personnel to find you.  If this form is found please contact Oak Grove Elementary School.  Name: Aiden.  Parent or Guardian: Mommy Daddy.  Phone number where you can be reached: I don't remember I think it has a 5 in it.  Teacher: Miss daisy.  Current location: I dont no I’m scared, Im hear all alone.  Medical Conditions i.e diabetes: I dont no what that is.  Additional Information: Why is everyone acting so weird?!\""
       }
     ]
+  },
```
Comment (Maleclypse) [inline data/json/snippets/children_notes.json:140]: You've added these below outside of the snippet object above.  Get rid of the `]` on line 139.

File: data/json/snippets/e_logs.json [hunk 3]
```diff
@@ -74,24 +74,56 @@
...
       },
       {
         "id": "local_files_simple_22",
         "text": "The photo depicts a serene and well-maintained terrace basked in warm, clear weather.  The central part of the picture is an intriguing sight, where a section of the terrace and a portion of the building behind it appear to have been cleanly removed, as if erased from reality.  To the side, there is a shattered flower pot, spilling out its soil and a plant with long, broad leaves, some of which have also been cut away in a sharp, clean fashion, almost as if sliced with a knife."
       }
     ]
```
Comment (Maleclypse) [inline data/json/snippets/e_logs.json:95]: Same problem with the closing bracket as above.  Delete line 95

**Review by Maleclypse — CHANGES_REQUESTED** [review]
Figured out why your tests were failing.

**Pull request merged**